src/schemas/listing-core.json

```diff
@@ -0,0 +1,243 @@
+        "region",
+        "country"
+      ]
+    },
+    "media": {
+      "type": "object",
+      "title": "Representation of a media asset such as an image or a video",
+      "properties": {
+        "url": {
+          "type": "string",
+          "format": "uri"
+        },
+        "contentType": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "dimension": {
+          "type": "object",
+          "properties": {
+            "height": {
+              "type": "integer"
+            },
+            "width": {
+              "type": "integer"
+            }
+          },
+          "required": [
+            "height",
+            "width"
+          ]
+        }
+      }
+    },
+    "timeSlot": {
+      "type": "object",
+      "required": [
+        "startDate",
+        "endDate",
+        "isAvailable"
+      ],
+      "properties": {
+        "startDate": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "endDate": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "isAvailable": {
+          "type": "boolean"
+        },
+        "price": {
+          "$ref": "#/definitions/moneyAmount"
+        }
+      },
+      "required": [
+        "startDate",
+        "endDate",
+        "isAvailable",
+        "price"
+      ]
+    },
+    "unitListing": {
+      "type": "object",
+      "properties": {
+        "unitsAvailable": {
+          "type": "integer"
+        },
+        "price": {
+          "$ref": "#/definitions/moneyAmount"
+        }
+      },
+      "required": [
+        "unitsAvailable",
+        "price"
+      ]
+    },
+    "fractionalListing": {
+      "type": "object",
+      "properties": {
+        "timeIncrement": {
+          "type": "string"
+        },
+        "calendarStep": {
+          "type": "string"
+        },
+        "slots": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/timeSlot"
+          }
+        }
+      }
+    },
+    "listingClass": {
+      "type": "object",
+      "title": "Support for Unit and Fractional listing types",
+      "properties": {
+        "name": {
+          "type": "string",
+          "enum": [
+            "unit",
+            "fractional"
+          ]
+        },
+        "fields": {
+          "anyOf": [
+            {
+              "$ref": "#/definitions/unitListing"
+            },
+            {
+              "$ref": "#/definitions/fractionalListing"
+            }
+          ]
+        }
+      },
+      "required": [
+        "name",
+        "fields"
+      ]
+    }
+  },
+  "type": "object",
+  "properties": {
+    "schemaVersion": {
+      "type": "string"
+    },
+    "class": {
+      "$ref": "#/definitions/listingClass"
+    },
+    "category": {
+      "type": "string"
+    },
+    "subCategory": {
+      "type": "string"
+    },
+    "locale": {
+      "type": "string"
```

Comment:
Instead of `locale`, I'd prefer `language` or `lang`, to match js and html conventions, respectively. While there is some debate on meanings of "local" vs "language"*, I feel we should just match javascript and go with `language`. 

*e.g. See "4. Locale versus Natural Language " section of https://www.w3.org/TR/ltli/

Reply:
Sounds good. Changed to _language_.